networking/libsnnet/network.go

```diff
@@ -19,11 +19,33 @@ package libsnnet
 import (
 	"fmt"
 	"net"
+	"reflect"
+	"sort"
 	"time"
 
 	"github.com/vishvananda/netlink"
 )
 
+// EqualNetSlice compare 2 network slices
+func EqualNetSlice(slice1, slice2 []string) bool {
+	equal := false
+	// if a slice is nil and other isn't then are not equal
+	// if length of slices are not the same, are not equal
+	if (slice1 == nil && slice2 != nil) ||
+		(slice2 == nil && slice1 != nil) ||
+		(len(slice1) != len(slice2)) {
+		equal = false
+	} else {
+		sortedSlice1 := make(sort.StringSlice, len(slice1))
+		sortedSlice2 := make(sort.StringSlice, len(slice2))
+		sortedSlice1.Sort()
+		sortedSlice2.Sort()
+		equal = reflect.DeepEqual(sortedSlice1, sortedSlice2)
```

Comment:
`return reflect.DeepEqual(sortedSlice1, sortedSlice2)` and you no longer need the `equal` variable.